.github/workflows/build-desktop.yml

```
@@ -13,6 +13,9 @@
 name: Build Desktop (reusable)
 on:
   workflow_call:
+permissions:
+  contents: read
+  actions: write
     inputs:
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)